File: Main/ArchiMeter.Analysis/TestData.fs
namespace ArchiMeter.Analysis

type TestData(requirementIds : seq<int>, assertCount : int, testName : string, testCode : string) =
    member this.RequirementIds = requirementIds
    member this.AssertCount = assertCount
    member this.TestCode = testCode
    member this.TestName = testName

File: Main/ArchiMeter.Analysis/TestData.cs
namespace ArchiMeter.Analysis
{
	using System.Collections.Generic;

	public class TestData
	{
		public TestData(IEnumerable<int> requirementIds, int assertCount, string testName, string testCode)
		{
			this.RequirementIds = requirementIds;
			this.AssertCount = assertCount;
			this.TestCode = testCode;
			this.TestName = testName;
		}

		public string TestName { get; private set; }
		
		public string TestCode { get; private set; }
		
		public int AssertCount { get; private set; }
		
		public IEnumerable<int> RequirementIds { get; private set; }
	}
}